dist/install-common.sh

```diff
@@ -31,14 +27,14 @@ force_debug_mode() {
 }
 
 wait_for_atfwd_daemon() {
-    until [ -n "$(adb shell 'pgrep atfwd_daemon')" ]
+    until [ -n "$ADB shell 'pgrep atfwd_daemon'" ]
```

Comment:
this won't run the command, it'll just always succeed cuz `"$FOO"` evaluates to a string w/ `$FOO` interpolated into it. you'll need:

```suggestion
    until [ -n "$($ADB shell 'pgrep atfwd_daemon)'" ]
```